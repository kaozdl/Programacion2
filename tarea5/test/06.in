# Caso 06
ingresar_localidades (50,ulsan) .
iniciar_pedidos 10
reiniciar_sistema
ingresar_localidades (50,ulsan) .
iniciar_pedidos 10
ingresar_pedidos ulsan (1000,chapas) 4 .
imprimir_totales
procesar_rango 1
procesar_rango 3
imprimir_totales
procesar_rango 3
ingresar_pedidos ulsan (1000,chapas) 4 .
imprimir_totales
ingresar_pedidos ulsan (1000,chapas) 4 .
imprimir_totales
ingresar_pedidos ulsan (1000,chapas) 1 .
imprimir_totales
ingresar_pedidos ulsan (1000,chapas) 9 ulsan (5000,ropa) 1 ulsan (2000,alimentos) 1 .
imprimir_totales
procesar_rango 1
procesar_rango 3
reiniciar_sistema
Fin
