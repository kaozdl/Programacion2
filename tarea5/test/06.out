1>2>3>
 IDENTIFICADORES DE LOCALIDADES 
(1,ulsan)
4>
  REINICIO DE SISTEMA 
5>6>
 IDENTIFICADORES DE LOCALIDADES 
(1,ulsan)
7>8>INFORME POR RANGOS
0
0
1000
0
0
9>10>ulsan
(1000,chapas)
11>INFORME POR RANGOS
0
0
0
0
0
12>ulsan

13>14>INFORME POR RANGOS
0
0
1000
0
0
15>16>INFORME POR RANGOS
0
0
2000
0
0
17>18>INFORME POR RANGOS
0
0
3000
0
0
19>20>INFORME POR RANGOS
0
0
11000
0
0
21>22>ulsan
(1000,chapas)(5000,ropa)(2000,alimentos)(1000,chapas)(1000,chapas)(1000,chapas)
23>
  REINICIO DE SISTEMA 
24>Fin.
